### DISCUSSION TITLE
Records regarding dialysis outside the ICU ward

### DISCUSSION THREAD

**Discussion by fysqy** [MIT-LCP/mimic-code#1709] [MIMIC-IV]
Does the mimiciv_derived rrt form, the mimiciv_public chartevent form, the mimiciv_public chartevent form, and the procedureevents form only record dialysis records while the patient is in the ICU? Can the patient's dialysis records in the ward be obtained before admission to the ICU or after discharge from the ICU?

**Comment by alistairewj** [comment]
Tables in the mimiciv_icu schema only pertain to the ICU, including *chartevents* and *procedureevents*. I believe the derived *rrt* table only uses these tables; but you can check the code to be sure.
